src/api/docs/content/pi-hole.js

```diff
@@ -101,13 +103,26 @@ function setStyle(style) {
 	docEl.setAttribute('render-style', style);
 	docEl.setAttribute('allow-search', style !== 'view');
 }
-document.addEventListener('DOMContentLoaded', (event) => {
-	let docEl = document.getElementById("thedoc");
 
-	docEl.addEventListener('after-try', (e) => {
-		console.log(e.detail.response);
-		if(e.detail.response.status === 401) {
-			loginFAIL();
+function setTheme(theme) {
+	const docEl = document.getElementById('thedoc');
+	docEl.setAttribute('theme', theme);
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+	const docEl = document.getElementById('thedoc');
+
+	docEl.addEventListener('after-try', (event) => {
+		console.log(event.detail.response);
+		if(event.detail.response.status === 401) {
```

Comment:
Unrelated to this patch, but should be addressed later, this throws an error. We should add try/catch here.